### ISSUE TITLE
Transparent header

### ISSUE THREAD

**Issue by HelenaDenley** [OllieWP/ollie#32]
Hello,

I've just downloaded your theme Ollie to try out on a community website I'm building. 

How would I go about making the header transparent and positioning it over the top of a background image or video?

I'm looking to get the same effect as this site - https://plast.org.au (it's using Kadence). 

Thank you 

Helena :)

**Comment by Ren2049** [comment]
Hey, 

I've [opened a bug report for the cover block](https://github.com/WordPress/gutenberg/issues/50610) earlier today, because that's the block best suited for this. 

Unfortunately it keeps the content always center, so it only works for very short transparent headers in a top bar style.

I don't think that there's a elegant way for taller transparent hero-headers like the one on your site so far.

Ideally they'd come up with a proper section block with media functionality similar to cover. I use background textures and patterns quite a lot and use mainly the cover block as a section wrapper, for most sections it works well enough. using the cover block is currently the only way to avoid a very basic looking site with the block editor imo.

**Comment by HelenaDenley** [comment]
On the scout site using Kadence, I have on the front page a row layout with the image as a background image and then in the header settings I have the header set to transparent and sticky (and it's just the height of the logo and nav) so when I scroll up, the background looks like it's scrolling behind the header. 
Obviously Kadence is still using the customiser and has settings that a straight block theme with full site editing doesn't have.

**Comment by Ren2049** [comment]
Kadence is way more mature and tuned to what pro bloggers need. The block editor has a lot of potential but the core devs don't have the domain knowledge to anticipate the needs of pro bloggers and site builders and don't have ways to communicate with the community like the Kadence devs have. 

The block editor is great if you geek out on efficiency and speed and are willing to make big compromises in layout and style of the site.

**Comment by HelenaDenley** [comment]
I may just need to return to Kadence for now. 
This site is for a community organisation that are not likely to have the funds to pay me, so the quickest I can get something good looking up for them the better. 
And I can come back to Ollie in the future for other projects.

**Comment by greening** [comment]
I'm guessing this is a valuable interaction for Mike McCallister
regardless, and definitely for me. Thanks Helena Denley.

**Comment by mikemcalister** [comment]
Hey folks, thanks for the feedback. Unfortunately, there's no easy way to do what @HelenaDenley is asking with the transparent header. There is a sticky header option, but without some more advanced mobile settings, it would be hard to replicate. 

Kadence may be better for this project, but keep an eye on the [Ollie Twitter](https://twitter.com/buildwithollie) page. This is just the beginning of this new era of WordPress, and the long term solutions will be very impressive.

@Ren2049 Let's keep the conversations about solutions! We don't need to throw anyone under the bus, including core devs. Making decisions for Kadence (thousands of users for a specific product) and making decisions for WordPress core (tens of millions of users for software that runs the web) are so drastically different that they can't be compared. I find myself frustrated as well, but I'm trying to channel that through innovation and creative problem solving with products like Ollie.

**Comment by HelenaDenley** [comment]
Thanks @mikemcalister - since returning to making websites after a 5 year break, I'm learning as I go with blocks and FSE and new themes like yours and I have no expectations for things to be perfect at this stage, so yes, I will definitely keep an eye on the Ollie Twitter page to keep updated with what is happening and I'm sure I'll use Ollie when I have the right project for it :)

**Comment by Ren2049** [comment]
@mikemcalister Sorry, it wasn't my intention to throw the devs under the bus. Kadence have their own forum where they directly collect feedback from users, the core devs don't, that's why they need us to relay those issues.  

>Making decisions for Kadence (thousands of users for a specific product) and making decisions for WordPress core (tens of millions of users for software that runs the web) are so drastically different that they can't be compared.

Sure, Is there a forum or place outside of GH where the core devs can pick the brains of block theme builders and designers? GH Issues get buried quickly..

I think the best way to adress millions is to create a working block theme economy. If you watch sites like Etsy or TF where WP themes are sold you still won't find any block theme there, I think because nobody takes the risk building on the current foundation.

I think it's important to let users know what classic theme users are getting into by adopting the block editor in its current state. I mean we still haven't collectively agreed upon what to put in templates at this point.  

A good blog post writing experience for example isn't possible without a narrow content width (setting content-width and wide width to 1260px for example), but if you set a narrow content width it makes building section patterns a pain because it's not wysiwyg when everything is compressed within the editor.

What I'm asking myself is why the core devs don't even regard this as an issue (they don't because if it would be already fixed)? I mean it's obvious if you design and write like people who'd use WP for blogging, funnels and to make a buck online.

**Comment by mikemcalister** [comment]
> the core devs don't, that's why they need us to relay those issues.

They do conduct some product research, I'm just not sure how often or how intense it is. There was a Gutenberg user working group at one point. But to your point, I think there could be a lot more focus on this (I'm just not sure how many are willing to donate their time to do it.)

> Sure, Is there a forum or place outside of GH where the core devs can pick the brains of block theme builders and designers? GH Issues get buried quickly

Yeah it's really tough to keep a robust convo going on Github sometimes. Twitter is pretty active for WordPress activity. And there is the WordPress core Slack channel as well. It's hard finding the time to contribute to all of the convos going on! 

> but if you set a narrow content width it makes building section patterns a pain because it's not wysiwyg when everything is compressed within the editor.

This shouldn't really be an issue. By default, the content area is narrower for post/page content, but group blocks come with the option to set a wide or full width of the page. That's how I built the many patterns in the Ollie theme, many of them use the full width group. I used the [Pattern Manager plugin](https://wordpress.org/plugins/pattern-manager/) to build them right inside the editor, and they look the same in the editor as they do the front end.

https://github.com/OllieWP/ollie/assets/1415737/d2177114-dc14-42a7-a454-953b5c9b59c8
